Entities/Items/Crate/Crate.as

```diff
@@ -328,6 +332,9 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream @params)
 			{
 				this.set_u32("unpack time", getGameTime() + this.get_u32("unpack secs") * getTicksASecond());
 				this.getShape().setDrag(10.0f);
+
+				CBlob @caller = getBlobByNetworkID( params.read_u16() );
```

Comment:
I hate this line so much 😂 
1. `@` goes right after the type, not before the variable name
2. Don't put whitespace around function arguments
```suggestion
				CBlob@ caller = getBlobByNetworkID(params.read_u16());
```